### PR TITLE
Spruce up OCSP code with our .clang-format

### PR DESCRIPTION
**Pull request by samuel40791765**
### Description of changes: 
Realigning indentation of OCSP code and adjusting documentation for docs.go.

### Call-outs:
Adjusted with CLion settings

### Testing:
N/A

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and 
the ISC license.

### CODE REVIEW

**Review by skmcgrail — APPROVED** [review]
Looks good CLion's diff makes it easier to condense the review since you can have it ignore the trivial whitespace changes.

File: include/openssl/ocsp.h [hunk 1]
```diff
@@ -140,45 +144,52 @@ OPENSSL_EXPORT int OCSP_request_add1_cert(OCSP_REQUEST *req, X509 *cert);
...
+//          code will not be passed if |*status| is NULL.
+OPENSSL_EXPORT int OCSP_resp_find_status(OCSP_BASICRESP *bs,
+                                         OCSP_CERTID *id,
+                                         int *status,
+                                         int *reason,
+                                         ASN1_GENERALIZEDTIME **revtime,
+                                         ASN1_GENERALIZEDTIME **thisupd,
+                                         ASN1_GENERALIZEDTIME **nextupd);
+
+// OCSP_basic_verify verifies a basic response message. Returns 1 on success,
```
Comment (andrewhop) [inline include/openssl/ocsp.h:173]: Does success == valid? Or just that the function ran to completion and you need to call something like OCSP_is_valid?
```suggestion
// OCSP_basic_verify verifies a basic response message. Returns 1 if the response is valid,
```

File: include/openssl/ocsp.h [hunk 2]
```diff
@@ -140,45 +144,52 @@ OPENSSL_EXPORT int OCSP_request_add1_cert(OCSP_REQUEST *req, X509 *cert);
...
+// 0 on error, or -1 on fatal errors such as malloc failure.
```
Comment (andrewhop) [inline include/openssl/ocsp.h:174]: Is this error the same as invalid signature?

File: include/openssl/ocsp.h (hunk 1 above)
Comment (samuel40791765) [reply include/openssl/ocsp.h]: Yeah, It test's that it's a valid response according to the certs passed in. Next step in a normal OCSP pipeline would to be check that the response hasn't expired.

File: include/openssl/ocsp.h (hunk 2 above)
Comment (samuel40791765) [reply include/openssl/ocsp.h]: Yes, clarified :)

**Review by samuel40791765 — COMMENTED** [review]
> CLion's diff 
TIL. Just tried it, the interface is much cleaner

**Comment by samuel40791765** [comment]
Turns out our .clang-format uses two space indents, so I've used that instead to align with the rest of the codebase

**Review by andrewhop — APPROVED** [review]

**Review by skmcgrail — APPROVED** [review]

**Pull request merged**